test-cypress/support/sparql-commands.js

```diff
@@ -0,0 +1,65 @@
+Cypress.Commands.add('pasteQuery', (query) => {
```

Comment:
What about reusing them in the sparql spec? Or at least create an issue so we won't forget.

Reply:
This is was WIP branch AFAIR. I'll update where necessary.